torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7400,6 +7427,14 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                   dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
                   supports_out=False,
                   sample_inputs_func=sample_repeat_tile),
+    OpInfo('trapz',
```

Comment:
Would you add a TODO comment here that in the future `trapz` should be made a proper alias of `trapezoid`

Reply:
There are two notable differences between `trapz` and `trapezoid`:
1. `dx` is Scalar for `trapezoid` and double for `trapz`
2. (This may not be different) We added `OpInfo` for both functions and they support complex tensor inputs

Given 1., can `trapz` still be an alias for `trapezoid`?

Reply:
I added the TODO comment for now nonetheless.